config.md

```diff
@@ -143,3 +143,41 @@ clang's parser, such as incoming references.
 Whether files are built in the background to produce a project index.
 This is checked for translation units only, not headers they include.
 Legal values are `Build` (the default) or `Skip`.
+
+## ClangTidy
+
+Configure how clang-tidy runs over your files.
+
+The settings are merged with any settings found in .clang-tidy
+configiration files with these ones taking precedence.
```

Comment:
do you mean checks specified via clangd configurations take precedence ?

Reply:
The checks and check options from clangd config take precedence over the clang tidy config files. Is that not obvious from the comment??

Reply:
that's what i assumed too but `these` was ambiguous right after talking about `.clang-tidy config files`. maybe `s/these ones/the ones from clangd configs/`.

Also s/configiration/configuration/